front/src/store/modules/eating.store.js

```diff
@@ -0,0 +1,48 @@
+import api from '@api';
+import { SERVER_SCHEDULED_PLACE_TO_EAT_PATH } from '@constants';
+
+const { baseUrl } = require('@/config/config.dev.json');
+
+export default {
+  namespaced: true,
+  actions: {
+    getEatingUser ({ commit }) {
+      return api.get('/scheduledPlaceToEat');
+    },
+    getEatingUserByEatingId ({ commit }, payload) {
+      return api.get('/scheduledPlaceToEat/' + payload.id)
+        .then(response => {
+          const scheduledPlaceToEat = response.data;
+          if (scheduledPlaceToEat.fileNames.length > 0) {
+            scheduledPlaceToEat.files = [];
+            scheduledPlaceToEat.fileNames.forEach(fileName => {
+              scheduledPlaceToEat.files
+                .push(`${baseUrl}/${SERVER_SCHEDULED_PLACE_TO_EAT_PATH}/${fileName}`);
+            });
```

Comment:
it should be in mapping dto logic - not at front

Reply:
ok